crmsh/bootstrap.py

```diff
@@ -840,11 +838,16 @@ def status(msg):
         print("  {}".format(msg))
 
 
+@contextmanager
 def status_long(msg):
     log("# {}...".format(msg))
     if not _context.quiet:
         sys.stdout.write("  {}...".format(msg))
         sys.stdout.flush()
+    try:
+        yield
+    finally:
+        status_done()
```

Comment:
To move the body of status_done() into the contex manager would make the code even nicer a little. 

Anyway, good Improvement!